src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -656,6 +674,8 @@ public void handle(AtmosphereResource r) {
         Message msg = CodecUtils.fromJson(bodyData, Message.class);
         msg.setProperty("uuid", uuid);
 
+        System.out.println(String.format(">>> %s", msg));

```

Comment:
comment out or use a proper log message

Reply:
ya .. was using it as high priority logging with log set to error level .. fixed in all instances inside webgui